config.go

```diff
@@ -20,6 +21,7 @@ const (
 	logLevelFlag       = optionPrefix + "log"
 	devModeFlag        = optionPrefix + "devmode"
 	traceModeFlag      = optionPrefix + "trace"
+	useVsockFlag       = optionPrefix + "use_vsock"
```

Comment:
We are now starting to document agent options in the top-level `README.md`. This isn't a general options like debug/trace but I wonder if we should add it anyway as it could be useful to someone?

Reply:
@jodh-intel Tests added. 
This option is something that should be added by the runtime, I dont think we should expose it to the user.

Reply:
makes sense @amshinde